.github/workflows/welcome.yml

```diff
@@ -0,0 +1,42 @@
+name: Welcome new contributors
+
+# This workflow has write permissions on the repo
+# It must not checkout a PR and run untrusted code
+
+on: pull_request_target
+
+jobs:
+  welcome:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/github-script@v6
+        with:
+          script: |
+            // Get a list of all issues created by the PR opener
+            // See: https://octokit.github.io/rest.js/#pagination
+            const creator = context.payload.sender.login
+            const opts = github.rest.issues.listForRepo.endpoint.merge({
+              ...context.issue,
+              creator,
+              state: 'all'
+            })
+            const issues = await github.paginate(opts)
+
+            for (const issue of issues) {
+              if (issue.number === context.issue.number) {
+                continue
+              }
+
+              if (issue.pull_request) {
+                return // Creator is already a contributor.
+              }
+            }
+
+            await github.rest.issues.createComment({
+              issue_number: context.issue.number,
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              body: `**Welcome**, new contributor!
+
+                Please make sure you're read our [contributing guide](CONTRIBUTING.md) and we look forward to reviewing your Pull request shortly ✨`
```

Comment:
Maybe something like an additional line:

"Thanks for helping Bevy grow! If you're not already there, we'd love to hear from you on our official Discord server <link here>, come say hi!"

Perhaps?

Reply:
Probably too noisy TBH :)

Reply:
I don't want to push discord that much. While it's active, it must not become mandatory for contributors...